docs/source/index.rst

```diff
@@ -1,7 +1,10 @@
-Python SDK
-==========
+Optimizely Python SDK
+=====================
 
-.. mdinclude:: ../../README.md
+.. toctree::
+   :caption: Introduction
+
+   readme
```

Comment:
There is no `readme.rst` yet right?

Reply:
There is, sorry! I just pushed it to the pr.